lib/keyboard_layouts.dart

```diff
@@ -115,6 +115,36 @@ const graphite = KeyboardLayout(
   ],
 );
 
+const galliumV2 = KeyboardLayout(
+  name: 'Gallium V2',
+  keys: [
+    ['B', 'L', 'D', 'W', 'Z', "'", 'F', 'O', 'U', 'J', ';', '='],
+    ['N', 'R', 'T', 'S', 'G', 'Y', 'H', 'A', 'E', 'I', ','],
+    ['Q', 'X', 'M', 'C', 'V', 'K', 'P', '.', '-', '/'],
+    [' '],
+  ],
+);
+
+const sturdy = KeyboardLayout(
+  name: 'Sturdy',
+  keys: [
+    ['V', 'M', 'L', 'C', 'P', "X", 'F', 'O', 'U', '-', '[', ']'],
```

Comment:
Replace '-' with J.

Reply:
You can check [ba0a267](https://github.com/conventoangelo/OverKeys/pull/1/commits/ba0a26751d1895b6e6665741de7b719c2e325efe), it's resolved.

Sorry about that, I used [oxey.dev](https://oxey.dev/playground/index.html) to see the layouts.